server/constants.js

```diff
@@ -19,17 +19,20 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-const PEERS_DEFAULT = '127.0.0.1:7933';
+// const PEERS_DEFAULT = '127.0.0.1:7933';
+const PEERS_DEFAULT = '127.0.0.1:5435';
 
 const REQUEST_RETRY_FLAGS_DEFAULT = { onConnectionError: true };
 
 const REQUEST_RETRY_LIMIT_DEFAULT = 3;
 
 const REQUEST_TIMEOUT_DEFAULT = 1000 * 60 * 5;
 
-const SERVICE_NAME_DEFAULT = 'cadence-frontend';
+// const SERVICE_NAME_DEFAULT = 'cadence-frontend';
+const SERVICE_NAME_DEFAULT = 'cadence-frontend-staging2';
```

Comment:
any chance we can pull this from envvars? it's leaning the Uber environment details out into the wild a bit.

Reply:
sorry I missed this. i will pull this out.

Reply:
also it does pull from env variables, these are just the defaults if none is provided.